aggregation.py

```diff
@@ -184,6 +185,7 @@ def _get_column_disclosure(self, df: pd.DataFrame, column_name: str) -> pd.DataF
 
             frequency_rule = (secrets['count'].sum() > self.frequency_threshold)
             dominance_rule = ((secrets['max'].max() * 100) < self.dominance_threshold * secrets['sum'].sum())
+            # vérification des règles <85% et plus de 3
```

Comment:
expliciter qu'est ce qui vérifie quoi :
```suggestion
            frequency_rule = (secrets['count'].sum() > self.frequency_threshold)
            # vérification de la règle "plus de 3" représentants
            dominance_rule = ((secrets['max'].max() * 100) < self.dominance_threshold * secrets['sum'].sum())
            # vérification qu'aucun ne représente >85% du total
```
Le terme représentant est le premier qui m'est venu, y'a sans doute mieux